pkg/dwarf/line/line_parser.go

```diff
@@ -33,8 +36,7 @@ type DebugLineInfo struct {
 	// lastMachineCache[pc] is a state machine stopped at an address after pc
 	lastMachineCache map[uint64]*StateMachine
 
-	// logSuppressedErrors enables logging of otherwise suppressed errors
-	logSuppressedErrors bool
+	log *logrus.Entry
```

Comment:
IMHO a reusable package like this shouldn't depend on a specific logging library that's not part of the standard library. Especially given the talk of contributing it to x/debug/dwarf.